cmd/ingester/main.go

```diff
@@ -79,8 +79,17 @@ func main() {
 		level.Error(util.Logger).Log("msg", "error initializing storage client", "err", err)
 		os.Exit(1)
 	}
+	var storageColumnKey chunk.StorageClient
+	if schemaConfig.BigtableColumnKeyFrom.IsSet() {
+		storageConfig.GCPStorageConfig.ColumnKey = true
+		storageColumnKey, err = storage.NewStorageClient(storageConfig, schemaConfig)
+		if err != nil {
+			level.Error(util.Logger).Log("err", err, "msg", "error initializing column key storage client")
+			os.Exit(1)
+		}
+	}
```

Comment:
Could this code be pushed into `chunk.NewStore` to avoid the repetition?

Reply:
Yep, I've done that first but then we need to import `.../chunk/storage` in `../chunk`, and `storage` imports `../chunk` causing a circular import.